src/javascripts/components/forms/addReservationForm.js

```diff
@@ -0,0 +1,127 @@
+import image from '../../helpers/images/seating.png';
+import reservationData from '../../helpers/data/reservationData';
+
+require('jquery-ui-bundle');
+
+const addGuestInfo = (data) => {
+  $('#add-reservation').html(`<h2>Enter User Info</h2>
+        <div id="success-message"></div>
+        <div>
+          <div id="error-message"></div>
+    <div id="input-group">
+        <div class="form-group">
+          <label for="image">First Name</label>
```

Comment:
I think this for label should match the id below it for "firstName". Same thing for the "lastName" and "phoneNumber" below. I think they were copied from somewhere else and we forgot to change them. 🙃